package/sbp_settings_daemon/src/settings.c

```diff
@@ -184,11 +184,11 @@ static void settings_read_reply_callback(u16 sender_id, u8 len, u8 msg[], void *
   const char *section = NULL, *setting = NULL, *value = NULL;
 
   if(!settings_parse_setting(len, msg, &section, &setting, &value, NULL))
-    log_error("Error in register message");
+    piksi_log(LOG_WARNING, "Error in register message");
```

Comment:
did you mean to take these from error's to warnings? Seems like there's a lot of errors becoming warnings here, just checking it was deliberate